eos/models/standard-model.cc

```diff
@@ -266,7 +266,7 @@ namespace implementation
     }
 
     double
-    SMComponent<components::QCD>::m_b_pole() const
+    SMComponent<components::QCD>::m_b_pole(const unsigned int loop_order) const
```

Comment:
The ``const`` here does not affect anything. ``unsigned int loop_order`` suffices completely!